### PR TITLE
Fix/#166

### LINKED ISSUE #166: Adjust Header Styles In Map  /  Ajustar Estilos De Encabezado En El Mapa
### Explanation:

When entering the text, it is bigger and in an inappropriate position.

**Current result:**
The text is too big and does not fit in the header.

**Expected result:** 

When entering the title text it must be in a proportional 16px size and aligned with a 20px space to the top left and the header with a size of 24%.

Resolution:
 (520X538) - (768x538)

### Screenshots:

image
image

Our:
image

### Checklist:

- [ ] Adjust text size in title and subtitle.
- [ ] Adjust header height: 24%.
- [ ] Align left on mobile screen
- [ ] Adjust line spacing.
- [ ] Enlarge header container, guided by layout.
- [ ] Check mobile view from 320X538 to 768X538.

### References:

https://www.figma.com/file/tbhMgDT10bbVjB9GazF97y/maps-proyect?node-id=407%3A46

---

# Spanish Version:

### Explicación:

Al ingresar el texto, es más grande y en una posición inapropiada.

**Resultado actual:**
El texto es demasiado grande y no cabe en el encabezado.

**Resultado Esperado:**

Al ingresar el texto del título debe estar en un tamaño proporcional de 16px y alineado con un espacio de 20px arriba a la izquierda y el encabezado con un tamaño de 24%.

Resolución:
 (520X538) - (768x538)

### Capturas de pantalla:

imagen
imagen

Nuestras:
imagen

### Lista de Verificación:

- [ ] Ajustar el tamaño del texto en título y subtítulo.
- [ ] Ajustar la altura del encabezado: 24%.
- [ ] Alinear a la izquierda en la pantalla del móvil.
- [ ] Ajustar interlineado.
- [ ] Ampliar contenedor de encabezado, guiado por diseño.
- [ ] Comprobar vista móvil de 320X538 a 768X538.

### Referencias:

https://www.figma.com/file/tbhMgDT10bbVjB9GazF97y/maps-proyect?node-id=407%3A46

### PR DESCRIPTION
**Pull request by mdrianislam0or1**
### Description
 Adjust Header Styles In Map / Ajustar Estilos De Encabezado En El Mapa #166 

#### Screenshots
fix#166 1
fix#166 2

### CODE REVIEW

File: src/components/Map/index.js [hunk 1]
```diff
@@ -1,19 +1,21 @@
-import React from 'react'
 import styled from '@emotion/styled'
-import PropTypes from 'prop-types'
 import GoogleMapReact from 'google-map-react'
-
+import PropTypes from 'prop-types'
+import React from 'react'
 import Marker from '../Marker'
 
+
```
Comment (Israel-Laguan) [inline src/components/Map/index.js]: Please remove this extra line

File: src/components/Map/index.js [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
 const Wrapper = styled.main`
   min-height: 356px;
   max-height: auto;
   max-width: var(--max-width);
   display: flex;
   flex-direction: row;
   flex: 1;
-  margin: 30px auto;
+  margin: 0px auto;
```
Comment (e-azocar) [inline src/components/Map/index.js]: you can remove `px`

File: src/pages/Delivery/Delivery.styled.jsx [hunk 3]
```diff
@@ -96,6 +96,10 @@ const HeaderSubTitle = styled.div`
   line-height: 28px;
   font-family: 'Poppins', sans-serif;
 
+  @media (max-width: 768px){
+  justify-content: left ; 
+  }
+
```
Comment (heliomar-pena) [inline src/pages/Delivery/Delivery.styled.jsx]: Please check indentation
```suggestion
  @media (max-width: 768px){
      justify-content: left; 
  }

```

**Review by Israel-Laguan — APPROVED** [review]

**Review by heliomar-pena — APPROVED** [review]

**Review by e-azocar — APPROVED** [review]

**Comment by angelik0828** [comment]
I think that the size of the text on the 320 mobile screen is not appropriate. There were some suggestions that had been made, in the issue an idea was given, in addition to the fact that the design shows texts that are more proportional to the screen sizes.


image


please check the checklist
image
image
image
image

**Review by angelik0828 — APPROVED** [review]

**Comment by arodriguezl9302** [comment]
@mdrianislam0or1 please resolve conflicts

**Pull request merged**